charts/pega/README.md

```diff
@@ -756,7 +756,43 @@ pegasearch:
     - name: TZ
       value: "EST5EDT"
 ```
+## Deploying Pega with external kafka for stream
```

Comment:
Can we provide an example or "getting started" configuration by default that will result in a functional system, as we do with Cassandra?